src/test/java/com/android/tools/build/bundletool/model/utils/files/FilePreconditionsTest.java

```diff
@@ -22,13 +22,15 @@
 import static com.android.tools.build.bundletool.model.utils.files.FilePreconditions.checkFileExistsAndReadable;
 import static com.android.tools.build.bundletool.model.utils.files.FilePreconditions.checkFileHasExtension;
 import static com.google.common.truth.Truth.assertThat;
+import static org.junit.Assume.*;
 import static org.junit.jupiter.api.Assertions.assertThrows;
 
 import com.android.tools.build.bundletool.model.utils.OsPlatform;
 import java.io.File;
 import java.nio.file.Files;
 import java.nio.file.Path;
 import java.nio.file.Paths;
+
```

Comment:
Remove extra line: https://google.github.io/styleguide/javaguide.html#s3.3.3-import-ordering-and-spacing